File: src/main/java/com/tmvkrpxl0/tcombat/common/enchants/CrossbowFlameEnchantment.java
package com.tmvkrpxl0.tcombat.common.enchants;

import net.minecraft.enchantment.Enchantment;
import net.minecraft.enchantment.EnchantmentType;
import net.minecraft.inventory.EquipmentSlotType;

public class CrossbowFlameEnchantment extends Enchantment {
    public CrossbowFlameEnchantment(Rarity p_i46737_1_, EquipmentSlotType... p_i46737_2_) {
        super(p_i46737_1_, EnchantmentType.CROSSBOW, p_i46737_2_);
    }

    public int getMinCost(int p_77321_1_) {
        return 20;
    }

    public int getMaxCost(int p_223551_1_) {
        return 50;
    }

    public int getMaxLevel() {
        return 1;
    }
}

File: src/main/kotlin/com/tmvkrpxl0/tcombat/common/enchants/CrossbowFlameEnchantment.kt
package com.tmvkrpxl0.tcombat.common.enchants

import net.minecraft.enchantment.Enchantment
import net.minecraft.enchantment.EnchantmentType
import net.minecraft.inventory.EquipmentSlotType

class CrossbowFlameEnchantment(rarityIn: Rarity, vararg slots: EquipmentSlotType) :
    Enchantment(rarityIn, EnchantmentType.CROSSBOW, slots) {
    override fun getMinEnchantability(enchantmentLevel: Int): Int {
        return 20
    }

    override fun getMaxEnchantability(enchantmentLevel: Int): Int {
        return 50
    }

    override fun getMaxLevel(): Int {
        return 1
    }
}